plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/db/ResourceGroupsDao.java

```diff
@@ -69,6 +69,7 @@
     List<SelectorRecord> getSelectors(@Bind("environment") String environment);
 
     @SqlUpdate("CREATE TABLE IF NOT EXISTS selectors (\n" +
+            "  id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,\n" +
```

Comment:
I think we should use https://flywaydb.org/ to handle the schema changes. Also this is very vendor specific syntax. I am not sure what RDBMS vendors we support here.

Reply:
See in-flight PR https://github.com/trinodb/trino/pull/9812 for Flyway.

I beleive we plan to support Postgres, MySQL and Oracle only.

Reply:
I have changed the syntax of primary key to make more generic. I don't know if it's a reasonable fix here, but i see resource_groups table also has similar primary key syntax.